Character.h

```diff
@@ -12,6 +12,7 @@ class Character {
 
 public:
     Character(const std::string, const int, const int);
+    static Character* parseUnit(const std::string);
     std::string getName() const;
```

Comment:
Itt is referencia legyen a string paraméter, illetve a getName se másolatot adjon vissza az adattagból, hanem const referenciát.

Reply:
A 2. fele még nincs meg.